service/gcs/core/gcs/storage.go

```diff
@@ -85,7 +122,7 @@ func (c *gcsCore) deviceIDToName(deviceID string) (string, error) {
 		// Devices matching the given SCSI code should each have a subdirectory
 		// under /sys/bus/scsi/devices/<scsiID>/block.
 		var err error
-		deviceNames, err = c.OS.ReadDir(filepath.Join("/sys", "bus", "scsi", "devices", scsiID, "block"))
+		deviceNames, err = ioutil.ReadDir(filepath.Join("/sys", "bus", "scsi", "devices", scsiID, "block"))
```

Comment:
I agree that we should phase out use of the oslayer interface in the future, and perform mocking for tests at a higher level. However, this change is not the time to do that. This will break the GCS unit tests.